### DISCUSSION TITLE
receive results from another template

### DISCUSSION THREAD

**Discussion by vagnerd** [projectdiscovery/nuclei-templates#7035] [Q&A] [answered]
Hello guys, there is any way to receive the results of another template?

Example: the file path found by the phpinfo-files module?

The idea is to use the url and path found by phpinfo-files to url request into another template.

**Comment by j4vaovo** [comment, marked as answer]
没必要考滤这些  一个模板实现即可 nuclei会自动合并相同的请求

**Reply by neouks** [reply]
你好师傅，有看到你在Nuclei Poc和observe的相关指纹库比较活跃，我最近也一直做这方面的事情，将漏洞转换成yaml方便自动化测试等，在各大群找了师傅好半天，我这边自己也在维护Templates和指纹，想交流一些这方面的知识或者一起做些这方面的事情，同时我也是知识星球Nuclei-Template的星主(人员不够20人公开搜索不到，**邀请制不圈钱不割韭菜)**，如果师傅有兴趣的话可以加我v: chainreactor

**Reply by j4vaovo** [reply]
@pphuahua 你发的是微信公众号吧 = =怎么加 我关注了啊, 看下公众号留言

**Reply by neouks** [reply]
不是公众号，是微信号，直接搜索微信号就好
@j4vaovo

**Comment by olearycrew** [comment]
You can do this with variables. Check out: https://nuclei.projectdiscovery.io/templating-guide/variables/ and https://nuclei.projectdiscovery.io/templating-guide/workflows/
